packages/build/src/commands/core_command.js

```diff
@@ -26,7 +26,6 @@ const fireCoreCommand = async function ({
       featureFlags,
       netlifyConfig,
     })
```

Comment:
This should use `|| {}` at the end, otherwise the following destructuring will fail:

https://github.com/netlify/build/blob/419e0d8ef1b68e5f955fbc7891e4acb11aaf4479/packages/build/src/commands/run_command.js#L71